A/AFL/build_tarballs.jl

```diff
@@ -0,0 +1,96 @@
+using BinaryBuilder, Pkg.Types
+
+name = "AFL"
```

Comment:
Only a question about the name: why isn't this called `AFLplusplus` like the upstream project?

Reply:
You are right, i should change this. My first attempts were using the original AFL sources, didn't change the name.

Reply:
Please also rename the directory to match exactly this name